### PR TITLE
20.7.1 Release Notes

### PR DESCRIPTION
**Pull request by jochman**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.7.1.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+The Threat Context module provides SOC, Incident Response, and Threat Intelligence teams with continuously updated and intuitive information around threat actors, campaigns, malware indicators, attack patterns, tools, signatures and CVEs.
+
+
+---
+
+### New: Zimperium Pack v1.0.0
+#### Classifiers
+##### Zimperium - Classifier  
+Classifies Zimperium incidents.
+#####
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.1.md:39]:
```suggestion
```

File: content-repo/extra-docs/releases/20.7.1.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+#### Integrations
+##### AWS Feed  
+Added the *Tags* parameter.
+
+---
+
+### Access Investigation Pack v1.1.4
+#### Layouts
+##### Access Incident  
+- Updated incident and indicator layouts to content pack format
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.1.md:69]:
```suggestion
Updated incident and indicator layouts to content pack format
```

**Comment by Itay4** [comment]
we need to be consistent with the bullets 
in some entities we add `-` before and some not

**Comment by jochman** [comment]
> we need to be consistent with the bullets
> in some entities we add `-` before and some not

We are adding only if there are multiple bullets.

**Comment by Itay4** [comment]
> > we need to be consistent with the bullets
> > in some entities we add `-` before and some not
> 
> We are adding only if there are multiple bullets.

we also have stuff like that
image

File: content-repo/extra-docs/releases/20.7.1.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+#### Integrations
+##### Have I Been Pwned? v2  
+Fixed an issue where the *test* button was not validating the API Key.
+
+---
+
+### Remedy AR Pack v1.0.1
+#### Integrations
+##### BMC Remedy AR  
+Updated the name of the integration to include the company name.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.1.md:317]: lets remove that

File: content-repo/extra-docs/releases/20.7.1.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+
+---
+
+### Remedy SR (Beta) Pack v1.0.1
+#### Integrations
+##### BMC Remedy SR (Beta)
+Updated the name of the integration to include the company name.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.1.md:324]: lets remove that

File: content-repo/extra-docs/releases/20.7.1.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+#### Integrations
+##### Workday  
+General performance and reliability improvements.
+
+---
+
+### Zimperium Pack v1.0.2
+#### Layouts
+##### Zimperium Event  
+Updated incident and indicator layouts to content pack format.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.1.md:387]: lets remove that

File: content-repo/extra-docs/releases/20.7.1.md [hunk 6]
```diff
@@ -0,0 +1,360 @@
+# Cortex XSOAR Content Release Notes for version 20.7.1 (68719)
+##### Published on 21 July 2020
+
+---
+
+Welcome to the 20.7.1 Content release for Cortex XSOAR. Starting from the 20.6.0 release, we restructured our release notes to be based upon **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Packs format were there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.7.1.md:6]: @kirbles19 we want to add to this disclaimer that we will still support installing content for earlier versions than 6.0
can you help please?

Comment (kirbles19) [reply content-repo/extra-docs/releases/20.7.1.md:6]: What's the method for installing content in pre-6.0 deployments?

Comment (Itay4) [reply content-repo/extra-docs/releases/20.7.1.md:6]: same as today, just through the ui

**Pull request merged**